packages/trpc/server/routers/viewer/webhook/edit.handler.ts

```diff
@@ -40,7 +43,12 @@ export const editHandler = async ({ input, ctx }: EditOptions) => {
     data,
   });
 
-  await updateTriggerForExistingBookings(webhook, webhook.eventTriggers, updatedWebhook.eventTriggers);
+  if (data.active) {
+    const activeTriggersBefore = webhook.active ? webhook.eventTriggers : [];
```

Comment:
If webhook was not active before, then there were no active triggers even when the webhook has triggers